samples/ConPTY/EchoCon/EchoCon/EchoCon.cpp

```diff
@@ -0,0 +1,207 @@
+// EchoCon.cpp : Entry point for the EchoCon sample application.
+//
+
+#include "stdafx.h"
+#include <Windows.h>
+#include <conio.h>
+#include <process.h>
+#include <wchar.h>
+
+void __cdecl PipeListener(LPVOID);
+HRESULT InitializeStartupInfoAttachedToConPTY(STARTUPINFOEX*, HPCON);
+COORD GetConsoleSize(HANDLE);
+
+int main()
+{
+    TCHAR szCommand[] = L"ping localhost";
+    HRESULT hr = S_OK;
+    HANDLE hPipeIn = 0;
+    HANDLE hPipeOut = 0;
+    HANDLE hPipePTYIn = 0;
+    HANDLE hPipePTYOut = 0;
+    HPCON hPC = 0;
+
+    PROCESS_INFORMATION piClient{};
```

Comment:
please standardize on one type of initialization; this is a hodge-podge of C++11 unified and old C style.